cypress/e2e/users_spec/UsersCreation.cy.ts

```diff
@@ -66,122 +66,87 @@ describe("User Creation", () => {
   });
 
   it("Update the existing user profile and verify its reflection", () => {
-    userCreationPage.clickElementById("user-profile-name");
-    userCreationPage.clickElementById("profile-button");
-    userCreationPage.verifyElementContainsText(
-      "username-profile-details",
-      "devdistrictadmin",
-    );
-    userCreationPage.clickElementById("edit-cancel-profile-button");
-    userCreationPage.typeIntoElementByIdPostClear(
-      "firstName",
-      "District Editted",
-    );
-    userCreationPage.typeIntoElementByIdPostClear("lastName", "Cypress");
-    userCreationPage.selectDropdownOption("gender", "Male");
-    userCreationPage.typeIntoElementByIdPostClear("phoneNumber", phone_number);
-    userCreationPage.typeIntoElementByIdPostClear(
-      "altPhoneNumber",
-      emergency_phone_number,
-    );
-    userCreationPage.typeIntoElementByIdPostClear("email", "test@test.com");
-    userCreationPage.typeIntoElementByIdPostClear("weekly_working_hours", "14");
+    userCreationPage.clickProfileName();
+    userCreationPage.clickProfileButton();
+    cy.verifyContentPresence("#username-profile-details", ["devdistrictadmin"]);
+
```

Comment:
remove this unwanted space

Reply:
https://github.com/AnveshNalimela/care_fe/blob/c4569fc16cb7f7f2d29552d660134c1868df9599/cypress/e2e/users_spec/UsersCreation.cy.ts#L93